sql/functionregistry.go

```diff
@@ -122,6 +122,9 @@ func (FunctionN) isFunction() {}
 // and User-Defined Functions.
 type FunctionRegistry map[string]Function
```

Comment:
Should this be changed to `type FunctionRegistry Functions`?

Reply:
Not really. They just happen to point to the same underlying type, but they have different purposes. `Functions` is just a shortcut and `FunctionRegistry` is the one with the methods and so on. I mean, it _could_ be changed, but it doesn't bring any gain, except that it may change in the future while `Functions` will probably not.